MediaGalleryUi/view/adminhtml/web/js/image/image-actions.js

```diff
@@ -75,16 +84,21 @@ define([
             var saveDetailsUrl = this.mediaGalleryEditDetails().saveDetailsUrl,
                 modalElement = $(this.modalSelector),
                 dataForm = modalElement.find('#image-edit-details-form'),
-                imageId = this.imageModel().getSelected().id;
+                imageId = this.imageModel().getSelected().id,
+                imageDetails = this.mediaGalleryImageDetails();
 
             if (dataForm.validation('isValid')) {
                 saveDetails(
                     saveDetailsUrl,
-                    dataForm
```

Comment:
We can leave the `dataForm` here instead, it is being used for validation and it is duplicating the code above.